### ISSUE TITLE
Confusion about configuring aliases in wiki

### ISSUE THREAD

**Issue by aminvakil** [tomav/docker-mailserver#1539]
## Aliases

* [x] I would like to contribute to the project (code, documentation, advocacy, integration, ...)

## Description

So I wanted to add some aliases for one of my mail servers (this image is great, currently migrating all of my mailservers to use this), said to myself RTFM, opened the wiki and there is an [aliases](https://github.com/tomav/docker-mailserver/wiki/Configure-aliases) page.

As the manual said I read the postfix documentation on virtual aliases, get some idea about how it works, and edit the `/tmp/docker-mailserver/postfix-aliases.cf`, adding this:
```
# Alias delivered to an existing account
alias1@domain.tld user1@domain.tld
```
I had created user1@domain.ltd beforehand and alias1@domain.tld didn't exist.
Now when I use another email trying to send an email to alias1@domain.tld, it always says:
```
said: 550
    5.1.1 <alias1@domain.tld>: Recipient address rejected: User unknown in
    virtual mailbox table (in reply to RCPT TO command)
```
Then I went and RTFM again, trying to forward all emails temporary to user1@domain.tld using this:
```
@domain.tld user1@domain.tld
```
Doesn't work again.
Long story short, after three days struggling with it, I found my "**best friend**":
https://github.com/tomav/docker-mailserver/wiki/Setup-docker-mailserver-using-the-script-setup.sh#alias
And it worked perfectly.

So I don't know if aliases wiki is wrong and doesn't work the way I have implemented it or I should have used `setup.sh`, and aliases wiki is just for some particular cases which I don't know what.

I'm willing to edit the wiki if it needs to change, but if it's fine I suppose it leads to misunderstanding in someones like me.

**Comment by erik-wramner** [comment]
Well, looking at the source code the setup script uses the file `/tmp/docker-mailserver/postfix-virtual.cf`, not `/tmp/docker-mailserver/postfix-aliases.cf`. I don't know why, the name in the wiki makes more sense but it does nothing.

Feel free to test manually editing that file and (if it works) update the wiki!

**Comment by aminvakil** [comment]
It uses `postfix-virtual.cf` indeed, I changed `postfix-aliases.cf` to `postfix-virtual.cf` in [wiki](https://github.com/tomav/docker-mailserver/wiki/Configure-Aliases) and add a link to setup.sh wiki page at top Configuring Aliases wiki page in case someone doesn't want to mess with files manually.
